### PR TITLE
add TravisCI integration

### PR DESCRIPTION
**Pull request by sprsquish**

### CODE REVIEW

File: src/model/Model.js [hunk 1]
```diff
@@ -380,7 +380,13 @@ export default class Model {
...
 /** Lambda expression for getting an instance Ref out of a match result. */
 function indexRefGetter(index) {
-  return index.values ?
-    arr => query.get(query.select(index.values.length, arr)) :
-    query.get
+  if (index.values) {
+    var idx = -1
+    for (var i = 0; i < index.values.length; i++) {
+      if (index.values[i]['path'] == 'ref') idx = i
+    }
```
Comment (sprsquish) [inline src/model/Model.js]: If values are defined on the index, refs are no long implicitly covered. They must be covered explicitly. Should we throw an exception here if the index isn't covering refs?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    - FAUNA_DOMAIN=localhost
+    - FAUNA_SCHEME=http
+    - FAUNA_PORT=8443
+jdk:
+  - oraclejdk8
+before_install:
+  - openssl aes-256-cbc -K $encrypted_c0ecaa32cf03_key -iv $encrypted_c0ecaa32cf03_iv -in core.tgz.enc -out core.tgz -d
+  - tar zxf core.tgz
+  - sh install-core.sh
+  - sudo apt-get install -y npm
```
Comment (andy-faunadb) [inline .travis.yml]: Usually installing node installs npm. Are you sure you need this?

Comment (sprsquish) [inline .travis.yml]: Node is pre-installed in the Travis environment. npm is not.

Comment (sprsquish) [inline .travis.yml]: Rather, there's no indication that I can see that it's installed so I'm
ensuring its there.

**Comment by andy-faunadb** [comment]
This should be ready to rebase and merge.

**Comment by andy-faunadb** [comment]
#60 is blocking this.

File: src/model/Model.js (hunk 1 above)
Comment (andy-faunadb) [inline src/model/Model.js]: Decided to put solving that problem on hold and removed the methods using this. See #60

**Pull request closed without merging**